src/app/shared/pipes/checkpoint-type.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { HelperService } from '../services/helper.service';
+
+@Pipe({
+  name: 'checkpointType',
+  standalone: true,
```

Comment:
Do you have an idea on if you want to make everything standalone at some point? Making this one standalone isnt bad but lets think about the future, maybe making everything standalone isnt a bad thing